.github/workflows/jaffle_shop/run_test_workflow.sh

```diff
@@ -6,6 +6,11 @@ dbt compile
 dbt run-operation stage_external_sources
 cp ../dbt-firebolt/.github/workflows/jaffle_shop/sources_external_tables_id_secret.yml models/staging/sources_external_tables.yml
 dbt run-operation stage_external_sources --vars "ext_full_refresh: true"
+if [[ -n "$AWS_ACCESS_ROLE_ARN" ]]; then
+    # Can't test this on FB 1.0
+    cp ../dbt-firebolt/.github/workflows/jaffle_shop/sources_external_tables_iam.yml models/staging/sources_external_tables.yml
+    dbt run-operation stage_external_sources --vars "ext_full_refresh: true"
```

Comment:
We already run this operation before this if block. Can we just execute it once after the if block?